src/status_im/chat/events.cljs

```diff
@@ -347,3 +343,19 @@
         (assoc :delete-chat chat-id)
         (not debug?)
         (assoc :deactivate-chat chat-id)))))
+
+(handlers/register-handler-fx
+  :delete-chat
+  [re-frame/trim-v]
+  (fn [cofx [chat-id]]
+    (-> (models/remove-chat cofx chat-id)
+        (update :db navigation/replace-view :home))))
+
+(handlers/register-handler-fx
+  :delete-chat?
+  [re-frame/trim-v]
+  (fn [_ [chat-id group?]]
+    {:show-confirmation {:title               (str (i18n/label :t/delete) "?")
```

Comment:
is "?" using in all languages?

Reply:
@jeluard ?

Reply:
Oh my bad, good point. Will make this label too.